di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/CoreStub.java

```diff
@@ -37,9 +37,17 @@ private void initRoutes() throws Exception {
         Spark.post("/edit-user", coreStubHandler.updateUser);
         Spark.get("/callback", coreStubHandler.doCallback);
         Spark.get("/answers", coreStubHandler.answers);
+        setupBackendRoutes(coreStubHandler);
         Spark.exception(Exception.class, exceptionHandler());
     }
 
+    private void setupBackendRoutes(CoreStubHandler coreStubHandler) {
```

Comment:
I think we should have an env var for this that is false by default so they are not available in the production env

Reply:
Variable added, default is enabled. A separate [PR](https://github.com/alphagov/di-ipv-config/pull/861) in stubs-config repo to disable these routes in pre-prod manifest.